lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -367,6 +367,23 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'If `false`, all LLM (OpenAI) instrumentation will be disabled and no metrics, events, or spans will be sent. AI Monitoring is automatically disabled if `high_security` mode is enabled.'
         },
+        :'ai_monitoring.record_content.enabled' => {
+          :default => true,
+          :public => true,
+          :type => Boolean,
+          :allowed_from_server => false,
+          :description => <<~DESCRIPTION
+            If `false`, LLM (OpenAI) instrumentation will not capture input and output content on specific LLM events.
+
+            The excluded attributes include:
+              * `content` from LlmChatCompletionMessage events
+              * `input` from LlmEmbedding events
+              * `input` from LlmTool events
```

Comment:
Yes, let's only reference events that we have at this time. We can update it if we end up supporting other events. 
LlmTool could be in our future since ruby-openai can access that part of the API.

Reply:
Thank you—Ill get these removed!